packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -156,4 +172,10 @@ export class SDKClient {
             ? input.substring(2)
             : input;
     }
+
+    private static toTinyBar(balance: AccountBalance): BigNumber {
+        return balance.hbars
+            .to(HbarUnit.Tinybar)
+            .multipliedBy(10_000_000_000);
```

Comment:
I assume that we have this constant used somewhere else. We can declare it as variable